internal/util/cli/styles.go

```diff
@@ -78,7 +79,13 @@ var (
 
 func init() {
 	// Get the terminal width, if available, and set widths based on terminal width
-	w, _, err := term.GetSize(int(os.Stdout.Fd()))
+	fd := os.Stdout.Fd()
```

Comment:
Admittedly this check is kind of pedantic since the max FD limit is well under 2^31 on any platform I've heard of.

Reply:
This is about getting the terminal width size which is used to format the CLI tables better, not the maximum amount of file descriptors on the OS

Reply:
oh, I missed expanding this and responded solely on the comment that you added. You kept the width so my comment is obsolete then 👍